README.md

```diff
@@ -116,6 +116,8 @@ You will probably need to run this for the `classes`, `subjects` and `reviews` c
 - **Michelle Li** - Developer Advisor
 - **Harrison Chin** - Developer
 - **Samuel Yeboah** - Developer
+- **Helen Lei** - Developer
```

Comment:
A minor thing: maybe specify that this PR is also adding people to the README or make it a separate PR